### DISCUSSION TITLE
requiredwith does not work with FileUpload

### DISCUSSION THREAD

**Discussion by Menks** [filamentphp/filament#12990] [Help]
Forms\Components\TextInput::make('t1')
                            ->requiredWith('t2'),

Forms\Components\FileUpload::make('t2')
                            ->requiredWith('t1') ,

**Comment by sathishg1990** [comment]
Yeah required validations are not working and can be resolved with FileUpload::make('t2')->required(fn (Get $get): bool => ! empty($get('t1')))
